README.md

```diff
@@ -3,50 +3,63 @@
-<br />
-</p>
-</details>
-<details><summary><b>✨ Provide a demo of your project</b></summary>
-<p>
-<br />
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-<br />
-</p>
-</details>
-<details><summary><b>👌 Insall/Usage must be super easy</b></summary>
-<p>
-<br />
-Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-<br />
-</p>
-</details>
-<details><summary><b>📘 Create a clean and well structured documentation</b></summary>
-<p>
-<br />
-TestLorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
-<br />
-</p>
-</details>
-<br />
-
-## 2. 📢 Spread the word about your hard works 
-<br />
-
-## 3. 🤝 Keep your users 
-<br />
\ No newline at end of file
+
+- 😎 Choose a cool name for your project
+
+- 💅 Make your README pretty
+
+- 👨‍🏫 Provide a demo of your project
+
+- 👌 Insall/Usage must be super easy
+
+- 📘 Create a clean and well structured documentation
+
+## 2. 📢 Spread the word about your hard works
+
+- ⭐ Talk about your project to your friends and colleagues to get your first stars before publishing to popular website
+
+    <details>
+    <summary>Why ?</summary>
+    <p>
+
+  > A minimum of stars make your project more trustable then a zero star project.
+
+    </p>
+    </details>
+
+- ↗️ Share the project on social medias an specialized platforms
+
+- ❓ Make a publication to explain the why and how of the project
+
+- 🕐 Take care to choose the best time to post and share articles
```

Comment:
I think this point should be placed just after `↗️ Share the project on social medias an specialized platforms` ?

Reply:
I hesistated to do that, but `❓ Make a publication to explain the why and how of the project` is also concerned by this point. An other solution it put this point before `↗️ Share the project on social medias an specialized platform` and `❓ Make a publication to explain the why and how of the project`. What do you think about that ?